azext_edge/edge/common.py

```diff
@@ -228,7 +258,7 @@ class FileType(ListableEnum):
     Supported file types/extensions for bulk asset operations.
     """
 
-    csv = "csv"
+    # csv = "csv"  # just use the portal format for ease
```

Comment:
does this commented csv still needed?

Reply:
yea I will get rid of it cause there is no point in trying to maintain two csv types